src/qbittorrent.exe.manifest

```diff
@@ -36,10 +36,12 @@
       </application>
   </compatibility>
 
-  <!-- Enable long paths that exceed MAX_PATH in length -->
-  <asmv3:application>
-    <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">
-      <longPathAware>true</longPathAware>
-    </asmv3:windowsSettings>
-  </asmv3:application>
+  <application xmlns="urn:schemas-microsoft-com:asm.v3">
+    <windowsSettings>
+      <!-- Enable Segment Heap, faster than NT Heap -->
```

Comment:
The "faster than NT Heap" somehow troubles me. I've seen some reports about it being slower: 
https://issues.chromium.org/issues/40138716
https://github.com/microsoft/Windows-Dev-Performance/issues/106

@Andarwinux 
Just curious, have you benchmarked qbt with it enabled?
You could just omit `faster than NT Heap` and we'll be fine.